llumnix/entrypoints/llumnix_utils.py

```diff
@@ -122,15 +122,21 @@ def init_llumlets(engine_manager_args: EngineManagerArgs,
     parallel_config = engine_config.parallel_config
     instance_ids: List[str] = []
     llumlets: List[Llumlet] = []
-    for _ in range(engine_manager_args.initial_instances):
-        instance_id = random_uuid()
+
+    instance_ids = [str(uuid.uuid4().hex) for _ in range(engine_manager_args.initial_instances)]
```

Comment:
We include random_uuid in llumnix utils.py in https://github.com/AlibabaPAI/llumnix/pull/11/. You can call random_uuid by importing it.